hhs_oauth_server/request_logging.py

```diff
@@ -527,7 +520,7 @@ def process_request(self, request):
         request._logging_uuid = str(uuid.uuid1())
         request._logging_start_dt = datetime.datetime.utcnow()
         request._logging_pass = 1
-        request._logger = RequestLogger(request)
+        request._logger = audit
```

Comment:
I think we can just remove this completely. Since we want to use the more specific names when possible, this will only be confusing to use.